packages/web3-eth-personal/src/rpc_method_wrappers.ts

```diff
@@ -83,7 +85,9 @@ export const signTransaction = async (
 	tx: Transaction,
 	passphrase: string,
 ) => {
-	const formattedTx = formatTransaction(tx, ETH_DATA_FORMAT);
+	const formattedTx = formatTransaction(tx, ETH_DATA_FORMAT, {
+		transactionSchema: requestManager.config.customTransactionSchema,
+	});
```

Comment:
i might be off here but the test failing on node 18 test/unit/eth_personal.test.ts  might pass if the third arg here is only passed in when `requestManager.config.customTransactionSchema` is truthy